src/implementations/vanilla/components/news-ticker/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@ecl/vanilla-component-news-ticker",
+  "author": "European Commission",
+  "license": "EUPL-1.2",
+  "version": "3.0.2",
```

Comment:
same as for the twig component, not really necessary, but you can align the version and the dependencies to 3.0.3